Data/IntSet.hs

```diff
@@ -73,10 +73,14 @@ module Data.IntSet (
 #endif
             , Key
 
-            -- * Operators
-            , (\\)
+            -- * Construction
+            , empty
+            , singleton
+            , fromList
+            , fromAscList
+            , fromDistinctAscList
 
-            -- * Query
+            -- * Queries
             , IS.null
             , size
             , member
```

Comment:
`member` goes at the top of the section. I don't think the section should be renamed like that.

Reply:
Done and done.